src/Uno.UI/UI/Xaml/Controls/TextBox/SinglelineTextBoxView.iOS.cs

```diff
@@ -112,11 +112,13 @@ public override bool BecomeFirstResponder()
 		{
 			var result = base.BecomeFirstResponder();
 
-			if (SecureTextEntry)
+			if (SecureTextEntry && !string.IsNullOrEmpty(Text))
 			{
 				var text = Text;
 				Text = string.Empty;
-				InsertText(text);
+				InsertText($"{text} ");
```

Comment:
Will this cause `PasswordChanging` and `PasswordChanged` to be raise? We might want to silence these events in this code path.

Reply:
Indeed, and if it does, we may need to have additional runtime tests for this.

Reply:
Good call. I will check and will post here.